.circleci/config.yml

```diff
@@ -107,6 +107,7 @@ commands:
             python -m pip install --user \
                 numpy<< parameters.numpy_version >> \
                 -r requirements/doc/doc-requirements.txt
+            pip install git+https://github.com/larsoner/sphinx-gallery.git@serial
```

Comment:
You can put this detail is the requirements files (which means it will hit all of the CI that pull it and any developers who try to build locally will also get it.

Reply:
On a general note: It's ok to use a non-released version for developing this fix. But this should not go into our main branch. We should wait with merging until an update of sphinx-gallery is released, and if necessary pin sphinx in the mean time.

Reply:
> You can put this detail is the requirements files (which means it will hit all of the CI that pull it and any developers who try to build locally will also get it.

moved sphinx-gallery installation to doc-requirements and all the checks passed :D